src/sidebar/components/Annotation/AnnotationBody.tsx

```diff
@@ -106,6 +106,7 @@ function AnnotationBody({ annotation, settings }: AnnotationBodyProps) {
             })}
             style={textStyle}
             mentions={annotation.mentions}
+            mentionsEnabled={store.isFeatureEnabled('at_mentions')}
```

Comment:
This is fine, though for performance I would recommend extracting a variable if the same selector needs to be called multiple times within a component.